### PR TITLE
Social suggestions

### PR DESCRIPTION
**Pull request by CoachGodzup**
Added a short guide on SEO and Social Sharing tips and tricks

### CODE REVIEW

**Review by Jean85 — APPROVED** [review]

File: how-to-write-posts.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- **LinkedIn sharing**: no description will be shown
+
+- **Messagging app** (Telegram, Slack, Whatsapp) will use Twitter information
+
+## SEO hints
+
+[We're working on automatizing this thing](https://github.com/facile-it/facile-it.github.io/issues/73), in the meantime I'll suggest you to **insert also a [JSON+LD](https://json-ld.org/) script in the page**, regarding the content of the article. You can see [this page for inspiration](content/blog/eng/v-protetto8-9-2023.md).
+
+In a tag `<script>` add a JSON object with all the information regarding your post. This will be used by Google crawler to get your content, and properly indexing it. *The better this data will be written, the greater visibility it gets*.
```
Comment (anaradujko) [inline how-to-write-posts.md:29]: I already incorporated these two sentences in the suggested paragraph above.

File: how-to-write-posts.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+`@context`: "https://schema.org",
```
Comment (anaradujko) [inline how-to-write-posts.md:31]: I would add a title here.

#### JSON-LS object format

File: README.md [hunk 3]
```diff
@@ -67,3 +67,8 @@ git commit [...] # to save it it
 `image`: used for sharing the URL in Facebook and LinkedIn. Best fit: 1600px x 900px. If not specified, the [ENGR logo](./static/images/social/social-preview.png) will be used;
 
 `twitterImage`: used for the twitter cards when sharing the url of this page (in X - former Twitter -, Telegram, Slack, Whatsapp); You can use the same image as `image` parameter. If not specified, the [ENGR logo](./static/images/social/social-preview.png) will be used;
+
+## Hints on how to write post
+
+they're included [in this article](./how-to-write-posts.md).
```
Comment (anaradujko) [inline README.md:73]: _they're included [in this article](./how-to-write-posts.md)._ -> The hints are listed [in this article](./how-to-write-posts.md).

**Review by anaradujko — COMMENTED** [review]
Hi! Ho dato un'occhiata e lasciato qualche commento. Spero siano d'aiuto. Se c'è qualcosa di poco chiaro, scrivimi pure.

**Comment by adamjesse01** [comment]
Unlock the secrets to skyrocketing your social media presence with our comprehensive guide on '[Social Media Growth](https://thetechnotricks.net) and Tricks.' Discover proven strategies, innovative hacks, and expert insights that pave the way for rapid expansion across various platforms. From understanding algorithms to leveraging engaging content and optimizing posting schedules, delve into the world of organic growth and savvy tactics. Learn how to captivate audiences, boost engagement, and build a thriving community while staying updated on the latest trends and tools.

File: how-to-write-posts.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+- **insert a property `title`**, on top of the article, Keep it short, for the sake of sharing. I suggest max 5 word length, longer titles will hide the description while sharing on Facebook.
+
+- **insert a short `description`**, tag that will be used on Facebook (cropped) and X. Keep this also short.
+
+- **add an image** in repo to be used as a **social sharing preview**. It should be fitted with the content. Best dimensions: 1600px x 900px. If in doubt, you can use [some default images](static/images/social/suggested/2.png). If omitted, [the default one will be used](static/images/social/social-preview.png).
+
+- custom twitter image should be **linked on top of the article**, as a `twitterImage` prop.
+
+- it's better also to **include the twitter image on top** of the article.
```
[the remark this answers is not in the document]
Comment (CoachGodzup) [reply how-to-write-posts.md:15]: Ok, I've mis-written. There are two "image" properties, one is TwitterImage (and uses Twitter metatag) and the other one is Image that uses OG metatags. It's better to have both

**Pull request merged**